From 6a5b4395db5153e9644224623701390820053659 Mon Sep 17 00:00:00 2001
From: Garrett Gu <garrett@cloudflare.com>
Date: Wed, 10 Apr 2024 14:31:33 -0500
Subject: Set torque generator path to external/v8. This allows bazel to find
 the generated torque outputs even though we are building from a different
 workspace.

See https://chromium-review.googlesource.com/c/v8/v8/+/5339896

diff --git a/bazel/defs.bzl b/bazel/defs.bzl
index a1539ab5399fe4759fcdbf93e62eb130cb7a2ae7..52b675a611e40246f19a96802b3acaeec4b47846 100644
--- a/bazel/defs.bzl
+++ b/bazel/defs.bzl
@@ -358,7 +358,7 @@ def _torque_files_impl(ctx):
     # when building V8 from a different repository and empty otherwise.
     v8root = ctx.label.workspace_root
     if v8root == "":
-        v8root = "."
+        v8root = "external/v8"
 
     # Arguments
     args = []
